src/main/java/com/whitedeath/organizationchart/View/Views.java

```diff
@@ -0,0 +1,13 @@
+package com.whitedeath.organizationchart.View;
+
+public class Views {
```

Comment:
Why is this class added?

Reply:
It is used to filter the fields of an entity. because in some output we don't have to show the managerId, so to ignore that field we used jsonview.
we can use jsonIgnore as well, but jsonIgnore is used for static filtering.
<img width="182" alt="Capture" src="https://user-images.githubusercontent.com/56544393/68273202-cdb70800-008b-11ea-8c10-333a0c1fe009.PNG">